### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by libbyhopfauf**
includes link to AMIA 2020 presentation video
link to blog posts and AMIA 2020 presentation video in "How to use"
added dvmerge
add dvguidance

### CODE REVIEW

**Comment by libbyhopfauf** [comment]
@dericed @ablwr @privatezero

File: README.md [hunk 1]
```diff
@@ -49,8 +51,13 @@ They are:
 
 **dvsampler** A script that downloads or creates DV files to use as samples. Requires FFmpeg.
 
+**dvmerge** A script that takes multiple transfers of the same tape containing errors and combines them to create one file with the best information available for each problematic frame.
```
Comment (dericed) [inline README.md:54]: The merge function is actually part of 'dvrescue'. See `dvrescue -h` on a recent build.

File: README.md [hunk 2]
```diff
@@ -49,8 +51,13 @@ They are:
 
 **dvsampler** A script that downloads or creates DV files to use as samples. Requires FFmpeg.
 
+**dvmerge** A script that takes multiple transfers of the same tape containing errors and combines them to create one file with the best information available for each problematic frame. 
+
+**dvguidance** A collection of additional resources including documentation, video demostrations, and solutions for how the operator may improve the transfer as guided by DVRescue.
```
Comment (dericed) [inline README.md:56]: links? :)

**Review by dericed — COMMENTED** [review]
couple comments, but glad this info is being added

**Comment by ablwr** [comment]
It would be good to link to https://mipops.github.io/dvrescue/ somewhere because I don't think it's obvious from the repo that it is up/available. And it does have some useful info!

**Comment by libbyhopfauf** [comment]
Definitely! I completely agree! Sorry, I meant to link to it in the dvguidance section.

File: README.md (hunk 1 above)
Comment (libbyhopfauf) [reply README.md]: Okay, gotcha. See new edits and let me know if that wasn't what you meant or if there were additional links that I missed.

**Review by dericed — APPROVED** [review]
LGTM, I'll squash-merge. Thx @libbyhopfauf!

**Pull request merged**